lms/views/error.py

```diff
@@ -52,12 +51,16 @@ def error(self):
 
 
     @view_config(context=MissingLTILaunchParamError)
-    def missing_lti_launch_param_error(self):
+    @view_config(context=MissingLTIContentItemParamError)
```

Comment:
Ideally this view could be a handler for `MissingLTILaunchParamError` only, if you adjusted the Exception hierarchy as I suggested in a previous comment. As `MIssingLTIContentItemParamError` inherits from `MissingLTILaunchParamError` in the suggested hierarchy, this view should "just work" for it.